### ISSUE TITLE
Consider expansion of range of asset classes

### ISSUE THREAD

**Issue by harrylandles** [JohnAdders/climate_narrative#107]
From Leo Donnachie,
Any plans to expand range of asset classes ?

**Comment by JohnAdders** [comment]
Yes we do plan to expand the list of asset classes, we do need an expert to step forward and offer to write up the tedxt for that asset class and then we can add it easily.
